sdk/forms/src/styles.css

```diff
@@ -0,0 +1,23 @@
+body {
```

Comment:
Why is this file here

Reply:
It's not packaged, it's just used for the local dev page to look nice.
Original thought was we might enrich this page to be a full demo but probably a demo app will be another project...